arbeitszeit_flask/flask_plotter.py

```diff
@@ -0,0 +1,25 @@
+import base64
+import io
+from datetime import datetime
+from decimal import Decimal
+from typing import List, Tuple
+
+from matplotlib.figure import Figure
+
+
+class FlaskPlotter:
+    def create_line_plot(
+        self, x: List[datetime], y: List[Decimal], size: Tuple[int, int] = (10, 5)
+    ) -> str:
+        fig = Figure()
```

Comment:
I think that the shortened variable names are making it harder to understand what is going on here.

Reply:
Changed `size` to `fig_size` - is that better? Regarding x and y: Not quite sure which name would make it more comprehensive. In plotting x and y are common variable names. Would you prefer `x_values` or `x_axes_values`?